lib/functions-filters.php

```diff
@@ -45,3 +45,37 @@ function add_lazysize_on_srcset($attr) {
 
 }
 add_filter('wp_get_attachment_image_attributes', 'add_lazysize_on_srcset');
+
+// This adds the height of the original image to the size in the srcset array that
+// matches the original size.
+//
+// This is added to the srcset in order to make lazysizes work with background images
+// via it's plugins bgset and parent-fit to work with background size cover.
+// Ref. https://github.com/aFarkas/lazysizes/tree/gh-pages/plugins/bgset
+//
+// Eg.
+// Array (
+//  [1552] => Array (
+//   [url] => http://localhost:8080/insearchofthetruth-net/wp-content/uploads/2017/08/e8b548419ee0ceaf94dbe9f8ce0a6f2f_original.jpg
+//   [descriptor] => h
+//   [value] => 1552w 873
+//   )
+// )
+//
+// NOTE: This is very hacky but works well and doesn't break anything
```

Comment:
are these your notes or from a code snippet somewhere? if so can you link it as another comment